docs/app/Examples/modules/Dropdown/Content/DropdownExampleDescription.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react'
 import { Dropdown } from 'semantic-ui-react'
 
+const options = [
+  { component: Dropdown.Header, icon: 'tags', content: 'Filter by tag' },
+  { component: Dropdown.Divider },
+  { description: '2 new', text: 'Important' },
+  { description: '10 new', text: 'Hopper' },
+  { description: '5 new', text: 'Discussion' },
+]
+
 const DropdownExampleDescription = () => (
-  <Dropdown text='Filter Tags' floating labeled button icon='filter' className='icon'>
-    <Dropdown.Menu>
-      <Dropdown.Header icon='tags' content='Filter by tag' />
-      <Dropdown.Divider />
-      <Dropdown.Item description='2 new' text='Important' />
-      <Dropdown.Item description='10 new' text='Hopper' />
-      <Dropdown.Item description='5 new' text='Discussion' />
-    </Dropdown.Menu>
-  </Dropdown>
+  <Dropdown text='Filter Tags' floating labeled button icon='filter' className='icon' options={options} />
```

Comment:
I think instead of overloading the `options` here, we use a new `items` prop.  There is quite a bit of logic based on the options for keyboard nav and selected/active/disabled states. Options is also meant to reflect the idea of `select` options.

My thinking with `items` is that is reflects any valid `Menu` item, including headers and dividers.  This will also play well with `<Menu items={} />` and `<Dropdown.Menu items={} />`.

Thoughts?